opentext-page.html

```diff
@@ -0,0 +1,501 @@
+<DOCTYPE html>
+<html>
+    <head>
+        <title>OpenText Page</title>
+        <style>
+            ul {
+                list-style-type: none;
+            }
+            li {
+                display: inline;
+            }
+            nav {
+                float: right;
+            }
+            .container {
+                float: left;
+                display: inline;
+            }
+            .col {
+                width: 16.6%;
+            }
+            a {
+                display: block;
+            }
+            .slide-show {
+                width: 50%;
+            }            
+            img {
```

Comment:
Although there is nothing technically wrong here (you can style HTML element tags directly like img, p, a), this selector targets ALL images in your html file.  The styles you have within this selector are width of 250px and float of left, which seems specific and wouldn't apply to all images.

In the future, unless you are targeting an entire group of tags, it's better to use classes or id's as the selector. 

No need to fix this here.  Just letting you know...